docs/concepts/alerts.md

```diff
@@ -0,0 +1,94 @@
+# Alerts and events
+
+Events are alerts are the vehicle for informing you about possible problem situations.
+
+- Event &mdash; something that happens in the system. An event may be a raw event (for example, drift-detection job detects drift in a given model) or a calculated event based on other events (CPU for a job is over 70% for one hour)
```

Comment:
The example of CPU over 70% may be a good idea, but it's not something that have been defined nor even close to being implemented in 1.7